File: example_3.py
from flask import Flask, render_template, request, jsonify
import requests

app = Flask(__name__)

@app.route('/')
def index():
    return render_template('index.html')

@app.route('/send_message', methods=['POST'])
def send_message():
    user_input = request.json['user_input']
    response = send_message_to_aelius(user_input)
    return jsonify({'response': response})

def send_message_to_aelius(user_input):
    url = 'https://aelius.live/api/v1/chat'
    headers = {'Content-Type': 'application/json'}
    data = {'user_input': user_input}
    
    try:
        response = requests.post(url, headers=headers, json=data)
        return response.json()['response']
    except Exception as e:
        print(f"Error fetching AI response: {e}")
        return "Sorry, I couldn't process that request."

if __name__ == '__main__':
    app.run(debug=True)

File: example_3.js
const express = require('express');
const fetch = require('node-fetch');
const bodyParser = require('body-parser');

const app = express();
app.use(bodyParser.json());

app.get('/', (req, res) => {
    res.sendFile(__dirname + '/index.html');
});

app.post('/send_message', async (req, res) => {
    const userInput = req.body.user_input;
    const response = await sendMessageToAelius(userInput);
    res.json({ response: response });
});

async function sendMessageToAelius(userInput) {
    const url = 'https://aelius.live/api/v1/chat';
    const response = await fetch(url, {
        method: 'POST',
        headers: {
            'Content-Type': 'application/json'
        },
        body: JSON.stringify({ user_input: userInput })
    });
    const data = await response.json();
    return data.response;
}

app.listen(3000, () => {
    console.log('Server started on http://localhost:3000');
});
